_drafts/oryx/technology.md

```diff
@@ -0,0 +1,31 @@
+# Oryx technologies
+
+Oryx is a frontend framework that is designed to help developers build web applications using modern technologies.
+It is built on a variety of technologies and in this article, we will take a closer look at each of these technologies and see how they contribute to the development of Oryx.
+
+## HTML
+HTML is the backbone of the web. It is the markup language used to create web pages and applications. In Oryx, HTML is used to structure the content and layout of the application. It provides a semantic structure to the page, making it easier to understand and maintain.
+
+## CSS
+CSS is a style sheet language used to describe the visual presentation of a document written in HTML. In Oryx, CSS is used to style the components and layout of the application. It allows developers to create a consistent visual style across the application and ensure that it is responsive and accessible.
+
+## JavaScript
+JavaScript is a high-level programming language that is widely used in web development. In Oryx, JavaScript is used to add interactivity and dynamic behavior to the application. It allows developers to create complex logic and manipulate the DOM in real-time.
```

Comment:
We can perhaps merge HTML, CSS and JS together, as it's the "web big three".  It should not have a lot of explanation.

Reply:
fair, but I can imagine we tackle that in the styling section. This is more high-level so happy with the changes you did.